ios/src/TGMapViewController.h

```diff
@@ -56,6 +56,7 @@ struct TileID;
 @property (assign, nonatomic) float zoom;
 @property (assign, nonatomic) float rotation;
 @property (assign, nonatomic) float tilt;
+@property (readonly, nonatomic) CGFloat pixelScale;
```

Comment:
nitpick, you should also declare `assign` here. I like to be explicit with memory management designation even though the compiler will figure out that CGFloat is a struct and should be assign anyway.